### PR TITLE
add possibility to hide smarty php notice messages from log

### PR DESCRIPTION
**Pull request by migoi**
## Description

Add possibility to hide Smarty's PHP NOTICE from phpError.log to easier detect and solve errors and warnings (see #4024 )
## Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code is written according to the guidelines found [here] (https://yetiforce.com/en/developer-documentation/standards/167-how-to-create-php-files.html).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: include/ConfigUtils.php [hunk 1]
```diff
@@ -86,7 +86,10 @@ public static function debug($key, $defvalue = false)
 			require_once 'config/debug.php';
 			AppConfig::load('debug', $DEBUG_CONFIG);
 		}
-		return self::$debug[$key];
+		if (isset(self::$debug[$key])) {
+			return self::$debug[$key];
+		}
+		return null;
```
Comment (mariuszkrzaczkowski) [inline include/ConfigUtils.php]: It should be false

Comment (mariuszkrzaczkowski) [inline include/ConfigUtils.php]: eg. return isset(self::$debug[$key]) ? self::$debug[$key] : false;

**Comment by migoi** [comment]
uhhhmm - don't like that short if's. Hard to read and source of some errors.

For the return value: I'm just orienting myself what is returned now in non existent case.

**Comment by mariuszkrzaczkowski** [comment]
whether standards are what define it?

**Pull request merged**